code/src/components/IntroPage.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+
+const IntroPage = ({ onStepChange }) => {
+	return (
+		<div className="welcome-message-container" tabIndex="0" id="first-question">
+			<h1 className="welcome-message">
+				Want to watch a horror movie?....................{' '}
+			</h1>
+
+			<button className="button" type="button" onClick={onStepChange}>
```

Comment:
As for one who doesn't like scary movies I would love to have the option to say no :D then maybe you could use and if/else and show not so scary movies for those who wants them.